core/evancore.vue.libs/src/interfaces.ts

```diff
@@ -74,6 +74,7 @@ export interface EvanVueOptionsInterface {
 export interface EvanFormControlUISpecs {
   attr?: {
     error?: string;
+    hint?: boolean|Function|string;
```

Comment:
```suggestion
    hint?: boolean | Function | string;
```
If it's known, add the function return type like `() => string` as well.

Reply:
Jeha it's correct. I will have a look at this stuff in future.